src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -895,36 +895,25 @@ namespace winrt::TerminalApp::implementation
     //   this stops us from reloading too many times or too quickly.
     fire_and_forget AppLogic::_DispatchReloadSettings()
     {
-        static constexpr auto FileActivityQuiesceTime{ std::chrono::milliseconds(50) };
-        if (!_settingsReloadQueued.exchange(true))
+        if (_settingsReloadQueued.exchange(true))
         {
-            co_await winrt::resume_after(FileActivityQuiesceTime);
-            _ReloadSettings();
-            _settingsReloadQueued.store(false);
+            co_return;
         }
-    }
 
-    fire_and_forget AppLogic::_LoadErrorsDialogRoutine()
-    {
-        co_await winrt::resume_foreground(_root->Dispatcher());
+        auto weakSelf = get_weak();
 
-        const winrt::hstring titleKey = USES_RESOURCE(L"ReloadJsonParseErrorTitle");
-        const winrt::hstring textKey = USES_RESOURCE(L"ReloadJsonParseErrorText");
-        _ShowLoadErrorsDialog(titleKey, textKey, _settingsLoadedResult);
-    }
-
-    fire_and_forget AppLogic::_ShowLoadWarningsDialogRoutine()
-    {
+        co_await winrt::resume_after(std::chrono::milliseconds(100));

```

Comment:
```suggestion
        co_await winrt::resume_after(std::chrono::milliseconds(50));
```
Shouldn't this be 50? because `FileActivityQuiesceTime` was originally 50?

Reply:
On my system 50ms wasn't always enough and the settings UI reloaded twice instead of just once.